pr_agent/tools/pr_description.py

```diff
@@ -73,6 +74,9 @@ def _prepare_pr_answer(self):
             markdown_text += f"## {key}\n\n"
             markdown_text += f"{value}\n\n"
         pr_body = ""
+        pr_types = []
+        if 'PR Type' in data:
+            pr_types = data['PR Type'].split(',')
```

Comment:
**Suggestion:** It's a good practice to use `isinstance()` instead of `type()` for type checking. `isinstance()` checks whether an object is an instance of a class or of a subclass thereof.
```suggestion
        pr_types = []
        if 'PR Type' in data:
            pr_types = data['PR Type'].split(',')
            if not isinstance(pr_types, list):
                pr_types = [pr_types]
```